### ISSUE TITLE
OTP-27 incompatibility

### ISSUE THREAD

**Issue by essen** [kafka4beam/snabbkaffe#63]
Hello! This project is not compatible with OTP-27.0-rc2 (and likely the upcoming full release). The problem comes from the use of `maybe` which has become a keyword.

```
src/snabbkaffe.erl:104:16: syntax error before: '::'
%  104| -type maybe(A) :: {just, A} | nothing.
%     |                ^

src/snabbkaffe.erl:126:35: syntax error before: '/'
%  126|              , maybe_pair/0, maybe/1, metric/0, run_config/0, predicate/0
%     |                                   ^
```

* https://github.com/kafka4beam/snabbkaffe/blob/master/src/snabbkaffe.erl#L104
* https://github.com/kafka4beam/snabbkaffe/blob/master/src/snabbkaffe.erl#L126

A quick fix could be to just put quotes around the word `'maybe'`.

**Comment by ieQu1** [comment]
Hello @essen,

Sorry for late response. I've just tagged `1.0.10` that supports OTP27. 

Side note: I had to introduce a rather ugly hack, though: https://github.com/kafka4beam/snabbkaffe/blob/master/include/common.hrl#L21 
I'll check if adding an explicit `maps:merge` affects the emitted code later, but right now snabbkaffe headers will turn off this new (stupid, IMHO) warning in any module that includes them.
